app/models/concerns/bulkrax/file_factory.rb

```diff
@@ -119,7 +121,7 @@ def import_files_filenames
       def destroy_existing_files
         return unless object.present? && object.file_sets.present?
         object.file_sets.each do |fs|
-          Hyrax::Actors::FileSetActor.new(fs, @user).destroy
+          Hyrax::Actors::FileSetActor.new(fs, user).destroy
```

Comment:
This fixed the issue seen in the screenshot. see: [user](https://github.com/samvera/bulkrax/blob/main/app/models/concerns/bulkrax/file_factory.rb#L41) 
![192014253-d586a10b-f36a-4080-a507-7668164696ad](https://github.com/samvera/bulkrax/assets/10081604/de9b1362-997b-4b8c-88f9-e58ab207fae9)